helm/private/helm_install.bzl

```diff
@@ -101,9 +228,20 @@ helm_uninstall = rule(
     implementation = _helm_uninstall_impl,
     executable = True,
     attrs = {
+        "helm_opts": attr.string_list(
+            doc = "Additional arguments to pass to `helm` during install.",
+        ),
         "install_name": attr.string(
             doc = "The name to use for the `helm install` command. The target name will be used if unset.",
         ),
+        "opts": attr.string_list(
```

Comment:
I think for all executables there's an implicit `args` attribute. Have you thought about using that for anything? I can see the use case for avoiding that attribute since I believe if you pass `args` to a target but then pass args on the command line the command line arguments override what you passed. Thus having a unique attribute is good for allowing invocation args to be appended.

Reply:
I wasn't really sure of the implications of using args for that reason. I could use that if you think it's better. If so, I'd also rename `helm_opts` to `helm_args`. Let me know what you want me to do.

Reply:
Naw I don't care about the name too much. Was mostly wondering if you'd thought about trying to use the existing plumbing.